src/framework/extensions/internal/legacy/extpluginsloader.cpp

```diff
@@ -93,15 +93,21 @@ Manifest ExtPluginsLoader::parseManifest(const io::path_t& path) const
     io::FileInfo fi(path);
 
     Manifest m;
-    m.uri = Uri("muse://extensions/v1/" + fi.baseName().toLower().toStdString());
+    m.uri = Uri("muse://extensions/v1" + path.toQString().sliced(rootPath.size()).toLower().toStdString());
```

Comment:
NB this does now mean the .qml extension is also included in the uri, which I would think is preferable anyway, case there are other possible extensions etc.